nautobot/extras/models/models.py

```diff
@@ -944,6 +953,187 @@ def log(
             logger.log(log_level, str(message))
 
 
+class ScheduledJobs(models.Model):
+    """Helper table for tracking updates to scheduled tasks.
+    This stores a single row with ident=1.  last_update is updated
+    via django signals whenever anything is changed in the ScheduledTask model.
+    Basically this acts like a DB data audit trigger.
+    Doing this so we also track deletions, and not just insert/update.
+    """
+
+    ident = models.SmallIntegerField(default=1, primary_key=True, unique=True)
+    last_update = models.DateTimeField(null=False)
+
+    objects = ExtendedManager()
+
+    @classmethod
+    def changed(cls, instance, **kwargs):
```

Comment:
Docstrings on these three functions explaining their usage (i.e. as signal handlers) would be appreciated.